tools/run_tests/run_xds_tests.py

```diff
@@ -2728,6 +2756,11 @@ def __init__(self, compute, alpha_compute, project, project_num):
                         client_process.returncode)
                 result.state = 'PASSED'
                 result.returncode = 0
+            except PotentialInfrastructureError as infra_error:
+                logger.exception('Test case %s skipped', test_case)
+                skipped_tests.append(test_case)
+                result.state = 'SKIPPED'
+                result.message = str(e)
```

Comment:
I was trying to figure out from the rendered functions how 'SKIPPED' results are rendered. Unless I'm missing something, the error message is hardcoded:
https://github.com/grpc/grpc/blob/86f93801e5d452cff40d6958d21a8eb211da0654/tools/run_tests/python_utils/report_utils.py#L119-L120

Might make sense to make a fake run and confirm if that hardcoded message `'Skipped'` needs to be replaced with `result.message` when it's not empty.